src/data/themes/dark.json

```diff
@@ -832,10 +878,10 @@
   "opacity": {
     "overlay": {
       "backdrop": {
-        "value": 0.8,
+        "value": "0.8",
```

Comment:
is this being a string gunna be an issue? 🤔

Reply:
Oof this was a mistake I made in the custom.js file, will be fixed to be a number when built